R/t-distribution/app.R

```diff
@@ -0,0 +1,163 @@
+library(shiny)
```

Comment:
so I actually forgot to push the progress I already made on the t-distribution app. Sorry!